modules.d/90crypt/module-setup.sh

```diff
@@ -33,20 +33,18 @@ installkernel() {
     [[ $hostonly ]] || [[ $mount_needs ]] && {
         # dmsetup returns s.th. like
         # cryptvol: 0 2064384 crypt aes-xts-plain64 :64:logon:cryptsetup:....
-        dmsetup table | while read name _ _ is_crypt cipher _; do
-            [[ $is_crypt != "crypt" ]] && continue
+        dmsetup table | while read -r name _ _ is_crypt cipher _; do
+            [[ $is_crypt == "crypt" ]] || continue
```

Comment:
We might want to document this preferred? evaluation method someplace ( coding style )

`[[ $is_crypt != "crypt" ]] && continue` 
vs
 `[[ $is_crypt == "crypt" ]] || continue`

Reply:
yeah.. positiv testing is preferred here

```
 [[ $is_crypt == "crypt" ]] || continue
```
as it catches the error case here.
`$?` is zero for the following commands

or
```
if [[ $is_crypt != "crypt" ]]; then
    continue
fi
```